bin/dm_redcap_scan_completed.py

```diff
@@ -141,27 +141,35 @@ def main():
     dir_meta = cfg.get_path('meta')
 
     # configure redcap variables
-    api_url = cfg.get_key(['REDCAP_URL'])
+    api_url = cfg.get_key('REDCAP_URL')
     redcap_url = api_url.replace('/api/', '/')
 
-    token_path = os.path.join(dir_meta, cfg.get_key(['REDCAP_TOKEN']))
+    token_path = os.path.join(dir_meta, cfg.get_key('REDCAP_TOKEN'))
     token = read_token(token_path)
 
-    redcap_project = cfg.get_key(['REDCAP_PROJECTID'])
-    instrument = cfg.get_key(['REDCAP_INSTRUMENT'])
+    redcap_project = cfg.get_key('REDCAP_PROJECTID')
+    instrument = cfg.get_key('REDCAP_INSTRUMENT')
 
     redcap_version = get_version(api_url, token)
 
     response = get_records(api_url, token, instrument)
 
     project_records = []
     for item in response.json():
+        status_val = item[cfg.get_key('REDCAP_STATUS_VALUE')]
+
+        #make status_val into a list
+        if not (isinstance(status_val,list)):
+            status_val=[status_val]
+
         # only grab records where instrument has been marked complete
-        if not (item[cfg.get_key(['REDCAP_DATE'])] and
-                item[cfg.get_key(['REDCAP_STATUS'])] == '1'):
+        if not (item[cfg.get_key('REDCAP_DATE')] and
+                item[cfg.get_key('REDCAP_STATUS')] in status_val):
```

Comment:
Just to double check, are there any cases where a T1 wasn't collected but the other scans were? Would just checking for the presence of `REDCAP_STATUS` or the `mr_status` variable for OPT be enough?

```suggestion
                item[cfg.get_key('REDCAP_STATUS')]):
```

Reply:
Jerry and I were discussing the best way to do this - we think that a T1 being collected is a more reliable indicator of scans being collected because the mr_status value hasn't been filled out by RAs in a consistent way.